test/bigtest/network/config.js

```diff
@@ -1,5 +1,10 @@
-// typical mirage config export
+import { inflections } from 'inflected';
+
 export default function config() {
+  inflections('en', function (inflect) {
+    inflect.irregular('campus', 'campus');
```

Comment:
Please add comment for the logic behind this.

Reply:
Done.